### PR TITLE
MAUI-2428-Updated Sliders and SignaturePad images and contents

### PR DESCRIPTION
**Pull request by PraveenGopalsamy**
Hi Team,

I have done the following changes in the Sliders and SignaturePad UG documentation. Please check the changes and merge it.
* Updated NuGet and create project images.
* Removed `Zero` and used `0` instead.
* Resolved the content alignment issues.
* Added API links for the Vol2 2022 features and controls.
* Added the tooltip always show property and also its customizing properties in the UG documentation. The new contents are reviewed by the content team. https://syncfusion.atlassian.net/browse/DOC-4522
* Updated the word `Grey` to `Gray` in the code snippets.

Regards,
Praveen G.

### CODE REVIEW

File: MAUI/DateTime-Range-Selector/tooltip.md [hunk 1]
```diff
@@ -59,11 +59,60 @@ rangeSelector.Content = chart;
...
+SfCartesianChart chart = new SfCartesianChart();
+rangeSelector.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![RangeSelector show always tooltip](images/tooltip/show-always-tooltip.gif)
+
+N> Refer [here](https://help.syncfusion.com/maui/datetime-range-selector/events-and-commands#tooltip-text-format) to customize the tooltip text format through the DateTime Range Selector events.
```
Comment (VijayakumarMariappan) [inline MAUI/DateTime-Range-Selector/tooltip.md:111]: Content is wrong. Update it as per discussed

File: MAUI/Range-Slider/divider.md [hunk 2]
```diff
@@ -185,8 +185,8 @@ Change the state of the range slider to disabled by setting `false` to the `IsEn
                         <VisualState.Setters>
                             <Setter Property="DividerStyle">
                                 <Setter.Value>
-                                    <sliders:SliderDividerStyle ActiveFill = "grey"
-                                                                    InactiveFill="LightGrey"
+                                    <sliders:SliderDividerStyle ActiveFill = "gray"
```
Comment (VijayakumarMariappan) [inline MAUI/Range-Slider/divider.md:188]: gray - Gray

**Review by VijayakumarMariappan — CHANGES_REQUESTED** [review]

File: MAUI/DateTime-Range-Selector/tooltip.md (hunk 1 above)
Comment (PraveenGopalsamy) [reply MAUI/DateTime-Range-Selector/tooltip.md]: Reupdated the content as dicussed.

File: MAUI/Range-Slider/divider.md (hunk 2 above)
Comment (PraveenGopalsamy) [reply MAUI/Range-Slider/divider.md]: Updated the word casing as mentioned.

**Comment by PraveenGopalsamy** [comment]
rebuild

**Review by VijayakumarMariappan — APPROVED** [review]
Changes are fine

**Pull request merged**